Dockerfile

```diff
@@ -7,19 +7,25 @@ ARG DEBIAN_FRONTEND=noninteractive
 ARG swisnap_repo=swisnap
 
 RUN apt-get update && \
-    apt-get -y install software-properties-common && \
-    LC_ALL=C.UTF-8 add-apt-repository -y ppa:rmescandon/yq && \
-    apt-get update && \
-    apt-get -y upgrade && \
+    apt-get upgrade -y && \
     apt-get -y install \
       apt-transport-https \
       ca-certificates \
+      gnupg \
       docker.io \
-      curl \
-      yq && \ 
+      curl && \
       apt-get clean && \
       rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+RUN arch="$(uname -m)" && if [ "${arch}" = "aarch64" ]; then \
+      yq_arch=arm64; \
+    elif [ "${arch}" = "x86_64" ]; then \
+      yq_arch=amd64; \ 
+    fi && \ 
+    curl -L "https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_${yq_arch}" -o yq && \
```

Comment:
yolo? why not from apt?

Reply:
no arm version

Reply:
There is no arm yq version released by rmescandon in ppa:rmescandon APT repository.